ui/plugin-system/src/components/DatasourceSelect.tsx

```diff
@@ -137,20 +137,27 @@ export function DatasourceName(props: { name: string; overridden?: boolean; over
 // Delimiter used to stringify/parse option values
 const OPTION_VALUE_DELIMITER = '_____';
 
-// Given a DatasourceSelector, returns a string value like `{kind}_____{name}` that can be used as a Select input value
-function selectorToOptionValue(selector: DatasourceSelector): string {
+/**
+ * Given a DatasourceSelectItemSelector,
+ * returns a string value like `{kind}_____{group}_____{name}` that can be used as a Select input value.
+ * @param selector
+ */
+function selectorToOptionValue(selector: DatasourceSelectItemSelector): string {
   return [selector.kind, selector.group ?? '', selector.name ?? ''].join(OPTION_VALUE_DELIMITER);
 }
 
-// Given an option value name like `{kind}_____{name}`, returns a DatasourceSelector
+/**
+ * Given an option value name like `{kind}_____{group}_____{name}`,
+ * returns a DatasourceSelector to be used by the query data model.
+ * @param optionValue
+ */
 function optionValueToSelector(optionValue: string): DatasourceSelector {
   const [kind, group, name] = optionValue.split(OPTION_VALUE_DELIMITER);
```

Comment:
it doesn't seem you are using the group here, or am I missing something ?

Reply:
We don´t use it but need to split still as it is in the optionValue 

I can do something like [kind, _, name] ? I don´t know if it works, let me try